datahandling.py

```diff
@@ -13,71 +13,58 @@
 import regex
 
 
-# methods to load files and filter data in them:
+def _load_pickle(path, encoding='utf-8'):
+    with open(path, mode="rb") as f:
+        try:
+            return pickle.load(f, encoding=encoding)
+        except UnicodeDecodeError:
+            f.close()
```

Comment:
With a context manager (`with open()...`) you don't need to (and indeed cannot) explicitly `.close()` -- the context manager does that for you, even when there is an error.